.github/workflows/atd_moped_cognito_hooks_build.yml

```diff
@@ -21,6 +21,7 @@ jobs:
     steps:
       - uses: actions/setup-python@v4
         with:
+          # this python version must be compatible with aws lambda!
```

Comment:
i tried bumping to 3.11 and ran into new issues. i think we could safely go up to 3.10 but i did not test it.